x-pack/test/case_api_integration/common/fixtures/plugins/observability/server/plugin.ts

```diff
@@ -21,6 +20,19 @@ export interface FixtureStartDeps {
   spaces?: SpacesPluginStart;
 }
 
+/**
+ * These are a copy of the values here: x-pack/plugins/cases/common/constants.ts because when the plugin attempts to
+ * import them from the constants.ts file it gets an error.
+ */
+const casesSavedObjectTypes = [
```

Comment:
You can only import from `common`, `server`, and `public`  modules. You could try exporting something from `common/index` and that might work for you.